src/helpers/GetData.js

```diff
@@ -9,6 +9,7 @@ class GetData {
       await element.scrollIntoView({ block: 'center', inline: 'center' });
       if (visible) {
         await Waiters.waitElementIsDisplayed(elementLocator);
+        var a;
```

Comment:
what is `var a`?

Reply:
Oh. yes. I am testing eslint rules and forgot about it.Thanks